### PR TITLE
add monorepo support to the CLI 

### PR DESCRIPTION
**Pull request by shairez**
# What is it?

- Feature / enhancement

# Description

added `projectDir` param to the cli, so the `add` command could generate files into a sub folder of a monorepo. 

plus, added `alwaysInRoot` prop to the integrations config on the CLI, to mark what should always stay in the root folder even when using the `--projectDir=` param.

 # Checklist

- [x] My code follows the [developer guidelines of this project](https://github.com/QwikDev/qwik/blob/main/CONTRIBUTING.md)
- [x] I performed a self-review of my own code
- [x] I added a changeset with `pnpm change`
- [x] I made corresponding changes to the Qwik docs
- [x] I added new tests to cover the fix / functionality

### CODE REVIEW

File: starters/features/drizzle/package.json [hunk 1]
```diff
@@ -22,7 +22,11 @@
         " Check out the Drizzle docs for more info:",
         "   - https://orm.drizzle.team/docs/overview"
       ]
-    }
+    },
+    "alwaysInRoot": [
+      "drizzle",
+      "drizzle.config.ts"
+    ]
```
Comment (wmertens) [inline starters/features/drizzle/package.json:29]: really? We have drizzle in Insights and it's in the insights package dir

File: starters/features/prisma/package.json [hunk 2]
```diff
@@ -20,7 +20,10 @@
         " Check out the Prisma docs for more info:",
         "   - https://www.prisma.io/docs/getting-started"
       ]
-    }
+    },
+    "alwaysInRoot": [
+      "prisma"
```
Comment (wmertens) [inline starters/features/prisma/package.json:25]: same

**Review by wmertens — DISMISSED** [review]
looking good but not entirely sure about some of the root files

LGTM

File: starters/features/drizzle/package.json (hunk 1 above)
Comment (shairez) [reply starters/features/drizzle/package.json:29]: good catch, thanks
fixing

**Review by wmertens — APPROVED** [review]
LGTM ⚡

**Pull request merged**